Plugin.pm

```diff
@@ -279,15 +279,44 @@ sub trackInfoMenu {
 	my ( $client, $url, $track, $remoteMeta ) = @_;
 	$remoteMeta ||= {};
 
+	my $isTidalTrack = $url =~ /^tidal:/;
 	my $extid = $track->extid;
-	$extid ||= $url if $url =~ /^tidal:/;
+	$extid ||= $url if $isTidalTrack;
 
 	my $artist = $track->remote ? $remoteMeta->{artist} : $track->artistName;
 	my $album  = $track->remote ? $remoteMeta->{album} : $track->albumname;
 	my $title  = $track->remote ? $remoteMeta->{title} : $track->title;
 
 	my $search = cstring($client, 'SEARCH');
 	my $items = [];
+
+	my $artists = ($track->remote and $isTidalTrack) ? $remoteMeta->{artists} : undef;
```

Comment:
make this an empty list ref rather than `undef`, or below `foreach` would throw an error

Reply:
👍 (Damn without a type checker or something similar you have to read carefully.)